third_party/xla/xla/stream_executor/rocm/rocm_blas.cc

```diff
@@ -983,7 +988,7 @@ bool ROCMBlas::DoBlasGemmBatched(
   const Eigen::bfloat16 beta_bf16(beta);
 
   tsl::Status status = DoBlasGemmBatchedInternal(
-      wrap::rocblas_gemm_strided_batched_ex, stream, transa, transb, m, n, k,
+      rocblas_gemm_strided_batched_bf16(), stream, transa, transb, m, n, k,
```

Comment:
may I ask what's differences with or without wrap? is only for dynamic loading? it seems many APIs are totally fine without wrap but it has existed in many places in our rocm API calls.